### PR TITLE
Bumped Kubernetes version to 1.19.15 and updated patches

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*
N/A

*Description of changes:*
* Bumped Kubernetes version to 1.19.15
* Updates existing patches (if kept)
* Improved commit message on existing patches
* Removed unneeded patches
* Added new patches
* Updated checksums

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/kubernetes/kubernetes/1-19/patches/0014-EKS-PATCH-Fix-kubectl-version-unit-test.patch [hunk 1]
```diff
@@ -0,0 +1,66 @@
+From 39bb1454ffa0c9b784b680fdadb955a64961c8aa Mon Sep 17 00:00:00 2001
+From: Eddie Zaneski <eddiezane@gmail.com>
+Date: Tue, 27 Jul 2021 11:46:45 -0600
+Subject: --EKS-PATCH-- Fix kubectl version unit test
+
+Cherry-pick of upstream Kubernetes:
+https://github.com/kubernetes/kubernetes/pull/103955, which was
+included in Kubernetes 1.22.
```
Comment (kschumy) [inline projects/kubernetes/kubernetes/1-19/patches/0014-EKS-PATCH-Fix-kubectl-version-unit-test.patch:8]:
```suggestion
included in Kubernetes 1.23.
```

**Comment by kschumy** [comment]
/hold

Might be adding new patch

**Review by jaxesn — APPROVED** [review]
/unhold
/lgtm

**Pull request merged**